handlers/handlers_test.go

```diff
@@ -75,10 +73,11 @@ func TestUnitHandlers(t *testing.T) {
 
 		Convey("test CreateFilterID returns 500 if unable to create filter job on filter api", func() {
 			mockClient := NewMockFilterClient(mockCtrl)
-			mockClient.EXPECT().CreateJob(gomock.Any()).Return("", errors.New("no filter job for you"))
+			mockClient.EXPECT().CreateBlueprint(gomock.Any(), gomock.Any()).Return("", errors.New("no filter job for you"))
```

Comment:
Error messages should also be updated: `errors.New("unable to create filter blueprint")`

Not sure why you have `for you` at the end of the message

Reply:
Because it's a test and the error message never gets checked, but I will change it just for you ;)